src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -423,7 +425,9 @@ export default class GeBatchGiftEntryTable extends LightningElement {
     }
 
     getColumn(element, fieldWrapper) {
-        const isReferenceField = element.dataType === 'REFERENCE';
+        const isReferenceField = element.dataType === 'REFERENCE' &&
```

Comment:
@kenlewis-sfdo Can we assign this `REFERENCE` string to a `const` variable instead of using the string directly?

Reply:
👍🏽